### ISSUE TITLE
[Feature request] Plugin to watch the training on web (tensorboard integrated with cogitare plugins)

### ISSUE THREAD

**Issue by aron-bordin** [cogitare-ai/cogitare#2]
- plot training error\std
- plot validation error\std
- time remaining
- button to stop the training process
- button to save the model at the current step
- button to pause the training
- button to resume the training
- plot model parameters statistics 
- save/load model execution log, to compare and analyze different executions [1]
- plot execution graph
- maybe something like named-scope from tensorflow [2]
- x-axis: by value or by relative time [3]
- plot smothing
- display real-time execution machine/gpu stats
- add Hyper-parameter option to modify its value from the web interface

[1] 
![screenshot from 2017-10-31 17-04-09](https://user-images.githubusercontent.com/4960137/32244041-71f1dd7c-be5e-11e7-91ca-e29b27a2578d.png)

[2] 
![screenshot from 2017-10-31 17-13-13](https://user-images.githubusercontent.com/4960137/32244151-cee6b39a-be5e-11e7-93cd-8e0cebdc5eda.png)

[3] 
![screenshot from 2017-10-31 17-52-29](https://user-images.githubusercontent.com/4960137/32245869-563cf6d8-be64-11e7-9067-bcff6b1669f4.png)

**Comment by aron-bordin** [comment]
moved to https://github.com/cogitare-ai/monitor
